cylc/flow/scripts/trigger.py

```diff
@@ -29,10 +29,25 @@
   # start a new "flow" by triggering 1234/foo
   $ cylc trigger --reflow my_flow//1234/foo
 
-Note: waiting tasks that are queue-limited will be queued if triggered, to
-submit as normal when released by the queue; queued tasks will submit
-immediately if triggered, even if that violates the queue limit (so you may
-need to trigger a queue-limited task twice to get it to submit immediately).
+Triggering a task queues it to run (Cylc internal queues) regardless of its
+prerequisites.
+
+Triggering a queued task causes it to submit immediately, regardless of queue
+limit.
+
+An "active task" is already submitted or running. Triggering an active task is
+dangerous: multiple instances of the same job could fight over the same Cylc
+log files at run time (and possibly over task-specific data files too).
```

Comment:
Doesn't triggering active tasks get ignored?
```
INFO - [1/foo submitted job:01 flows:1] => running
INFO - [1/foo running job:01 flows:1] health: execution timeout=None, polling intervals=PT15M,...
INFO - Processing 1 queued command(s)
WARNING - [1/foo running job:01 flows:1] ignoring trigger - already active
```

Reply:
Oops, you're right. Had it in my head that we still allowed that, in case the user knows better than the scheduler.

Reply:
Fixed, and mentioned triggering incomplete tasks too.